_team/developers/01-roel.md

```diff
@@ -0,0 +1,8 @@
+---
+name_: Roel Algaba Brizuela
+description: Software engineer
```

Comment:
Not sure if German wants to show here that he is in a **Jr** position.

Reply:
I spoke with German, Software Engineer is fine